app/controllers/users_controller.rb

```diff
@@ -4,6 +4,7 @@ class UsersController < ApplicationController
   before_action :set_user, only: %i[update update_twitch_username update_language_settings confirm_destroy request_destroy full_delete remove_association]
   after_action :verify_authorized, except: %i[index signout_confirm add_org_admin remove_org_admin remove_from_org]
   before_action :authenticate_user!, only: %i[onboarding_update onboarding_checkbox_update]
+  before_action :redirect_to_lowercase_route, only: %i[index]
```

Comment:
I'm a bit confused here. I don't think the `index` route is the right place to be checking this. The `index` route is being used for follow and sidebar suggestions, right? I also tried to QA this locally and put a `binding.pry` and never actually hit `users#index`.

When I look at the routes, it looks like when we make a GET request to `/:username` we should be going through `stories#index`, and internally, that method calls `handle_user_index`, which is where we're actually finding the user by the passed in `params[:username]`.

Reply:
Also, for more context, the way that I was QAing this is by going to a local user's page and then changing their user name to something that was _not_ downcased (for example, `http://localhost:3000/CartwrightFrancesco`. Do let me know how you were QAing this locally and if I should be doing something different, but I'm assuming this is the behavior/request that we're trying to catch!

Reply:
Huh, it looks like we actually _do_ downcase usernames [here](https://github.com/thepracticaldev/dev.to/blob/master/app/controllers/stories_controller.rb#L197) 🤔

Reply:
Yeah, `users#index` here is actually `dev.to/users`, an API endpoint that returns json as supposed to `dev.to/ben` which is in stories.

Reply:
Okay, these are all good points and this should definitely live in the `Stories::Controller`--I should have known that. 🤦‍♀️ I've since moved this logic there.

Huh, that is strange...

And to answer your question about QAing, the way that you've been QAing my work is exactly how I've been QAing it as well.

Reply:
> Yeah, `users#index` here is actually `dev.to/users`, an API endpoint that returns json as supposed to `dev.to/ben` which is in stories.

lol seems like a good candidate for removal @maestromac :D Going to bring it up for team OSS